english/you-are-in-minecraft/minecraft.md

```diff
@@ -0,0 +1,19 @@
+You open your eyes and there you are surrounded by a crafting table, a furnace,
```

Comment:
It looks like this pull request has the new file, but not the change to `marshmallow.md`, and the other pull request has the opposite problem. If you can get both changes into the same pull request, you should be good!